packit_service/worker/helpers/testing_farm.py

```diff
@@ -295,7 +301,10 @@ def _payload_without_token(payload: Dict) -> Dict:
         return payload_
 
     def _construct_fmf_payload(self) -> dict:
-        fmf = {"url": self.fmf_url}
+        fmf = {
+            "url": self.fmf_url,
+            "path": self.fmf_path,
+        }
```

Comment:
What about having it optional and not sending anything if not configured? (Similar to the fmf_ref option.)

Reply:
(But I see it's needed to check if the FMF is configured.)

Reply:
So far this has worked without it on testing-farm side, altgough the api appears to be outdated either on this side or theirs (`tmt` vs `fmf` naming).

We are doing a fmf testing on packit side, so might be better for debugging down the line to be explicit.